File: 1 - Two sum/main.rs
impl Solution {
    pub fn two_sum(nums: Vec<i32>, target: i32) -> Vec<i32> {
        for (i, &num1) in nums.iter().enumerate() {
            for (j, &num2) in nums.iter().enumerate() {
                if i != j && num1 + num2 == target {
                    return vec![i as i32, j as i32];
                }
            }
        }
        vec![]
    }
}

File: 1 - Two sum/main.py
from typing import List

def twoSum(nums: List[int], target: int) -> List[int]:
    for i in range(len(nums)):
        for j in range(len(nums)):
            if i != j and nums[i] + nums[j] == target:
                return [i, j]
    return []


if __name__ == "__main__":
    my_list = [2, 4, 6]
    print(twoSum(my_list, 6))
